docs/guide.rst

```diff
@@ -86,3 +86,141 @@ options::
 
 Go on and explore. Don't be shy and ask, ``--help`` is eager to
 answer all your questions ;-)
+
+
+Under The Hood
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now let's have a brief look under the hood. Similarly as with
+``git``, there is a special ``.fmf`` directory which marks root of
```

Comment:
Missing article in front of root -> `the root`

Reply:
Thanks, added.